common/pb/moderator_commands.proto

```diff
@@ -17,3 +18,10 @@ message Command_BanFromServer {
     optional string visible_reason = 5;
     optional string clientid = 6;
 }
+
+message Command_GetBanHistory {
+    extend ModeratorCommand {
+        optional Command_GetBanHistory ext = 1001;
+    }
+    optional string user_name = 1;
```

Comment:
When working with the protocol it's important to consider what it makes possible, not just the current client behavior. In about 5 minutes I can write a client that can send any specific command with any variation of parameters. If some permutation isn't handled and leads to a crash, I've got a DOS script. Anyone could do this.

> What was your acronym again? Something about not writing code for the future or trying to predict the future.... i dont recall. :)
YAGNI - and I agree.

So let's keep the field as optional and then have the server reject the command if it's not present.